ThirdParty/frozen/bits/pmh.h

```diff
@@ -96,13 +96,15 @@ pmh_buckets<M> constexpr make_pmh_buckets(const carray<Item, N> & items,
   result_t result{};
   // Continue until all items are placed without exceeding bucket_max
   while (1) {
-    for (auto & b : result.buckets) {
-      b.clear();
-    }
     result.seed = prg();
     for (std::size_t i = 0; i < N; ++i) {
       auto & bucket = result.buckets[hash(key(items[i]), static_cast<size_t>(result.seed)) % M];
-      if (bucket.size() >= result_t::bucket_max) { continue; }
+      if (bucket.size() >= result_t::bucket_max) {
+        continue;
+        for (auto & b : result.buckets) {
+          b.clear();
+        }
```

Comment:
Isn't the `continue` going to skip the for loop? It also looks like if this is here than clearing all of the buckets shouldn't be happening multiple times.

Reply:
Hmm,  that would be a bug in the frozen library.   I will investigate that further.